src/mmw/js/src/draw/utils.js

```diff
@@ -84,15 +85,44 @@ function cancelDrawing(map) {
     map.fire('draw:drawstop');
 }
 
+function getGeoJsonLatLngs(shape) {
+    if (shape.coordinates) {
+        return L.GeoJSON.coordsToLatLngs(shape.coordinates, 2);
+    } else if (shape.geometry) {
+        return L.GeoJSON.coordsToLatLngs(shape.geometry.coordinates, 1);
+    } else if (shape.features) {
+        var coordinates = [];
+        _.forEach(shape.features, function(feature) {
+            coordinates.push(feature.geometry.coordinates);
+        });
+        return L.GeoJSON.coordsToLatLngs(coordinates, 2);
+    }
+    return null;
+}
+
 // Return shape area in km2.
 function shapeArea(shape) {
     return coreUtils.changeOfAreaUnits(turfArea(shape),
-            'm<sup>2</sup>', 'km<sup>2</sup>');
+        'm<sup>2</sup>', 'km<sup>2</sup>');
+}
+
+// Get the bounding box of the shape and return its area in km2
+function shapeBoundingBoxArea(shape) {
+    var shapeLatLngPoints = getGeoJsonLatLngs(shape),
+        latLngBounds = L.latLngBounds(shapeLatLngPoints),
+        boundingBox = [
+            latLngBounds.getWest(),
+            latLngBounds.getSouth(),
+            latLngBounds.getEast(),
+            latLngBounds.getNorth()
+        ],
+        boundingBoxPolygon = turfBboxPolygon(boundingBox);
+        return shapeArea(boundingBoxPolygon);
```

Comment:
🎨  outdent this line one level. Also consider having a blank space between variable declarations and `return` statement.